src/Common/Commands.Common/MetricHelper.cs

```diff
@@ -207,9 +207,11 @@ public void FlushMetric()
         /// <returns></returns>
         public static string GenerateSha256HashString(string originInput)
         {
-            SHA256 sha256 = SHA256.Create();
-            var bytes = sha256.ComputeHash(Encoding.UTF8.GetBytes(originInput));
-            return BitConverter.ToString(bytes);
+            using (var sha256 = new SHA256CryptoServiceProvider())
```

Comment:
@markcowl Can you please describe, how changing the algorithm class removes the exception.
Maybe I am not able to understand what was causing the exception in the first place.